x-pack/plugin/security/qa/security-trial/src/javaRestTest/java/org/elasticsearch/xpack/security/role/BulkPutRoleRestIT.java

```diff
@@ -181,15 +181,74 @@ public void testPutNoValidRoles() throws Exception {
     public void testBulkUpdates() throws Exception {
         String request = """
             {"roles": {"test1": {"cluster": ["all"],"indices": [{"names": ["*"],"privileges": ["all"]}]}, "test2":
-            {"cluster": ["all"],"indices": [{"names": ["*"],"privileges": ["read"]}]}, "test3":
-            {"cluster": ["all"],"indices": [{"names": ["*"],"privileges": ["write"]}]}}}""";
-
+            {"cluster": ["all"],"indices": [{"names": ["*"],"privileges": ["read"]}], "description": "something"}, "test3":
```

Comment:
Added some more explicit validation that we can reset some top level fields. These things are already implicitly caught by other tests but I think it's nice to do some more validation.